lib/i18n_rails_helpers/engine.rb

```diff
@@ -8,4 +10,13 @@ class Engine < Rails::Engine
       ActionView::Base.send :include, ListLinkHelpers
     end
   end
+
+  ActionController::Base.class_eval do
+    include ControllerHelpers
+  end
+
+  ActiveRecord::Base.class_eval do
+    include ModelHelpers
+    after_initialize :define_enum_t_methods
+  end
```

Comment:
Sorry, I meant put these inside the `initializer` block above. It probably doesn't actually make a difference but should be more future-proof.

Reply:
I was just not quite sure how. But I'll try and find out.

Reply:
You can move the code as it is actually, should work.